src/frontend/src/navigation/navigation.tsx

```diff
@@ -10,17 +11,38 @@ class Navigation extends React.PureComponent<Props> {
     render() {
         return (
             <NavigationContainer>
-                <Link to='/'>
-                    <span>testorino</span>
-                </Link>
+            <StyledLink to='/rezervacie' isActive={"/rezervacie"===this.props.path}>
+                    <span>Rezervacie</span>
+                </StyledLink>
+                <StyledLink to='/historia' isActive={"/historia"===this.props.path}>
+                    <span>Historia objednavok</span>
+                </StyledLink>
+                <StyledLink to='/klienty' isActive={"/klienty"===this.props.path}>
+                    <span>Sprava klientov</span>
+                </StyledLink>
+                <StyledLink to='/procedury' isActive={"/procedury"===this.props.path}>
```

Comment:
Ten path si s props vytiahni pred renderom
`const { path } = this.props;`
cize potom budes mat len
`isActive={'/historia' === path}`